.prettierrc

```diff
@@ -0,0 +1,13 @@
+{
+  "printWidth": 80,
+  "singleQuote": true,
+  "trailingComma": "none",
```

Comment:
Shall we use trailing comma? 😄 

The benefits are:
- Easier line movement (don't need to worry about missing comma)
- Better git blame (adding/removing comma adds noise to it)
- Multi-cursor in editor becomes more useful

Reply:
I'm +1 for trailing commas if you want to enable them